ql/src/test/queries/clientpositive/sharedwork_semi_2.q

```diff
@@ -0,0 +1,104 @@
+set hive.auto.convert.anti.join=false;
+set hive.auto.convert.join.noconditionaltask.size=1145044992;
+set hive.auto.convert.join.noconditionaltask=true;
+set hive.auto.convert.join=true;
+set hive.auto.convert.sortmerge.join.to.mapjoin=true;
+set hive.auto.convert.sortmerge.join=true;
+set hive.cbo.enable=true;
+set hive.convert.join.bucket.mapjoin.tez=true;
+set hive.enforce.sortmergebucketmapjoin=true;
+set hive.exec.dynamic.partition.mode=nonstrict;
+set hive.exec.dynamic.partition=true;
+set hive.exec.max.dynamic.partitions.pernode=4000;
+set hive.exec.max.dynamic.partitions=10000;
+set hive.exec.parallel.thread.number=32;
+set hive.exec.parallel=false;
```

Comment:
why do you need to set all these irrelevant options?

Reply:
removed irrelevant configurations